resources/qml/WelcomePages/AddNetworkOrLocalPrinterContent.qml

```diff
@@ -104,6 +104,20 @@ Item
         }
     }
 
+    // This "Back" button only shows in the "Add Machine" dialog, which has "back_button_text" set to "Cancel"
```

Comment:
This comment will get outdated FAST :grin: Better to describe when it should be shown and when not.

Reply:
Fixed in bb1442a04e1239693892cbea012bd6b9b7018727